lib/dalli/socket.rb

```diff
@@ -13,15 +13,7 @@ module Socket
     # Common methods for all socket implementations.
     ##
     module InstanceMethods
-      def readfull(count)
```

Comment:
We no longer need this method, but we still need `read_available` as it is used for the pipelined methods.